tests/wasm/sourcemap_compileunits.test

```diff
@@ -0,0 +1,30 @@
+# RUN: %yaml2obj %s -o %t.obj
```

Comment:
Thanks so much for adding yaml2obj tests. Could we add one for diff mode, so we can test our design for distinguishing between different versions of the same file?

Reply:
Thanks. I think the simplest way to do this would be to make a copy of the Wasm binary generated by yaml2obj, then create 2 different source maps for each of these. Made the change. PTAL